lib/execution-options.js

```diff
@@ -15,11 +17,37 @@ const proxyExecuteKey = "ProxyExecute";
  * </p>
  */
 class ExecutionOptions {
+    /**
+     * @type {_rust.QueryOptionsWrapper?}
+     */
+    #rustWrapper;
+
     /**
      * Creates a new instance of {@link ExecutionOptions}.
      */
     constructor() {}
 
+    /**
+     * Creates rust options wrapper for this execution options unless it already exists.
+     * Goal of this function is to avoid wrapping the same options multiple times.
+     * @package
+     */
+    wrapOptions() {
+        if (!this.#rustWrapper) {
+            this.#rustWrapper = queryOptionsIntoWrapper(this);
+        }
+    }
```

Comment:
```suggestion
    /**
     * Creates rust options wrapper for this execution options unless it already exists.
     * Goal of this function is to avoid wrapping the same options multiple times.
     * @package
     */
    wrapOptionsIfNotWrappedYet() {
        if (!this.#rustWrapper) {
            this.#rustWrapper = queryOptionsIntoWrapper(this);
        }
    }
```